rpc_jobs/standard_job_premerge.yml

```diff
@@ -0,0 +1,134 @@
+- project:
+    name: "gating"
+    repo_url: "https://github.com/rcbops/rpc-gating"
+    branches:
+      - "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - "default"
+    action:
+      - "deploy"
+    jobs:
+      - 'RPC_{name}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'RPC_{name}-{image}-{scenario}-{action}'
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "30"
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGION: "{REGION}"
```

Comment:
REGION is now REGIONS after RE-48 merged

Reply:
Bother, that means we can't inject the applicable region as an env var in the job any more. I'll have to remove it.